hips/utils/wcs.py

```diff
@@ -4,16 +4,20 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
+from astropy.coordinates import Angle
 from astropy.wcs.utils import pixel_to_skycoord, wcs_to_celestial_frame
 
 __all__ = [
     'WCSGeometry',
 ]
 
-__doctest_skip__ = ['WCSGeometry']
+__doctest_skip__ = [
+    'WCSGeometry',
+    'WCSGeometry.*'
```

Comment:
One alternative would be to use `*` in place of both `WCSGeometry` and `WCSGeometry.*` which would skip all the tests. Do you think that is more suitable?